app/pages/sections/consignee/ConsigneeSection.scala

```diff
@@ -33,15 +33,11 @@ case object ConsigneeSection extends Section[JsObject] {
       UpdateNeeded
     } else {
       (
-        request.userAnswers.get(ConsigneeExportPage),
+        request.userAnswers.get(ConsigneeExportInformationPage),
```

Comment:
This is currently in main so not sure if it should be fixed here but if I _select_ a VAT number but don't enter one then it sets the section as completed. Same with EORI.

Reply:
Fix will be made in ETFE-3748